src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 3.3.0
 -----
 
+ * Added `kernel.project_dir` and `Kernel::getProjectDir()`
+ * Deprecated `kernel.root_dir` and `Kernel::getRootDir()`
```

Comment:
Virtually at least :) We cannot really display a deprecation as it's used everywhere. But, I want to get rid of it in Symfony Flex, so that in 4, we can really deprecate `root_dir`.